avm/res/search/search-service/shared-private-link-resource/main.bicep

```diff
@@ -18,24 +18,24 @@ param groupId string
 param requestMessage string
 
 @description('Optional. Can be used to specify the Azure Resource Manager location of the resource to which a shared private link is to be created. This is only required for those resources whose DNS configuration are regional (such as Azure Kubernetes Service).')
-param resourceRegion string = ''
+param resourceRegion string?
```

Comment:
If updated here, you can also simplify it in the parent bus just passing the value through. No default in the parent is required as the property would be nullable

Reply:
Done: https://github.com/Azure/bicep-registry-modules/pull/1218/commits/19e6b5c29c74076c9cca5eb1754f074d9247a2e5